spring-boot/README.MD

```diff
@@ -6,4 +6,5 @@ The "REST With Spring" Classes: http://bit.ly/restwithspring
 - [Intro to Spring Boot Starters](http://www.baeldung.com/spring-boot-starters)
 - [A Guide to Spring in Eclipse STS](http://www.baeldung.com/eclipse-sts-spring)
 - [Introduction to WebJars](http://www.baeldung.com/maven-webjars)
+- [How to Register a Servlet in a Java Web Applicationt](http://www.baeldung.com/how-to-register-a-servlet-in-a-java-web-application/)
```

Comment:
Note: I will add this after the article is published, because we don't know what the final URL will be.